### ISSUE TITLE
Admin's Cannot Remove Portals at all

### ISSUE THREAD

**Issue by pokemonrocks9** [oddlama/vane#27]
We have a portal at spawn we wanted to customize and edit however we've noticed theres no way to edit or deactivate or even destroy the portal from the config. storage.json keeps re-adding the part for this portal as well. Is there a potential way to have a way so admins can edit a portal together or remove an abandoned portal perhaps?

**Comment by pokemonrocks9** [comment]
We have also had a users portal disconnect and be unable to be repaired like this

**Comment by oddlama** [comment]
Yes, you are right, it would be good to have a way for admins to remove a portal. I'll look into this when I have time.

Currently only the original builder of a portal may currently edit or destroy it. If it is inside a vane region, it may be edited by all admins of that region, but removal is still only possible by the owner. 

You can definitely remove it by editing the storage.json, but you need to be careful to actually catch __all__ occurrences of the portal, i.e. not only the portal definition with all blocks, but also all references to the portal uuid. You can also only do this while the server is offline (otherwise it will just be overwritten again, as you already experienced.)
